libcovebods/schema.py

```diff
@@ -49,6 +61,16 @@ def __work_out_schema_version(
         # If bad data passed, then we assume it's the default version
```

Comment:
This is now "assumes default version if not not record based, assumes latest version if record based" right? maybe comments in config.py could be updated too?

Reply:
Have improve comments here, and added more detail in config